Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -637,18 +637,39 @@ int CLuaPedDefs::GetPedTargetStart(lua_State* luaVM)
     return 1;
 }
 
+int CLuaPedDefs::OOP_GetPedTargetStart(lua_State* luaVM)
+{
+    CClientPed*      pPed = nullptr;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pPed);
+
+    if (!argStream.HasErrors())
+    {
+        CVector vecStart;
+        pPed->GetShotData(&vecStart);
+
+        lua_pushvector(luaVM, vecStart);
+        return 1;
+    }
+    else
+        m_pScriptDebugging->LogCustom(luaVM, argStream.GetFullErrorMessage());
```

Comment:
should be an error rather than warning

Reply:
What do you mean? All the `else` path for argStream errors that I saw so far are like this 🤔

Reply:
new functions should throw errors rather than log warnings - https://github.com/multitheftauto/mtasa-blue/blob/b9d509a6855c285df9439016604782bea1ae8440/Shared/mods/deathmatch/logic/luadefs/CLuaXMLDefs.cpp#L227